SpatialGDK/Source/SpatialGDK/Private/Tests/SpatialView/ViewCoordinatorTest.cpp

```diff
@@ -0,0 +1,220 @@
+﻿// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "ComponentTestUtils.h"
+#include "SpatialView/OpList/EntityComponentOpList.h"
+#include "SpatialView/OpList/ExtractedOpList.h"
+#include "SpatialView/ViewCoordinator.h"
+#include "Tests/TestDefinitions.h"
+
+#define VIEWCOORDINATOR_TEST(TestName) GDK_TEST(Core, ViewCoordinator, TestName)
+
+namespace SpatialGDK
+{
+// A stub for controlling the series of oplists fed into the view coordinator. The list of oplists given to
+// SetListsOfOplists will be processed one list of oplists at a time on each call to Advance.
+class ConnectionHandlerStub : public AbstractConnectionHandler
+{
+public:
+	void SetListsOfOpLists(TArray<TArray<OpList>> List) { ListsOfOpLists = MoveTemp(List); }
+
+	virtual void Advance() override
+	{
+		QueuedOpLists = MoveTemp(ListsOfOpLists[0]);
+		ListsOfOpLists.RemoveAt(0);
+	}
+
+	virtual uint32 GetOpListCount() override { return QueuedOpLists.Num(); }
+
+	virtual OpList GetNextOpList() override
+	{
+		OpList Temp = MoveTemp(QueuedOpLists[0]);
+		QueuedOpLists.RemoveAt(0);
+		return Temp;
+	}
+
+	virtual void SendMessages(TUniquePtr<MessagesToSend> Messages) override {}
+
+	virtual const FString& GetWorkerId() const override { return WorkerId; }
+
+	virtual const TArray<FString>& GetWorkerAttributes() const override { return Attributes; }
+
+private:
+	TArray<TArray<OpList>> ListsOfOpLists;
+	TArray<OpList> QueuedOpLists;
+	FString WorkerId = "test_worker";
+	TArray<FString> Attributes{ "test" };
```

Comment:
Nit - use = for this case to make it clear you mean to use the initialiser list constructor

Reply:
done